src/components/Sidebar.vue

```diff
@@ -109,20 +112,83 @@
 				</template>
 			</NcAppNavigationItem>
 		</template>
+		<template #footer>
+			<NcAppNavigationSettings :name="t('news', 'Settings')">
+				<NcButton @click="showHelp = true">
+					{{ t('news', 'Keyboard shortcuts') }}
+				</NcButton>
+				<HelpModal v-if="showHelp" @close="showHelp=false" />
+				<div>
+					<div>
+						<input id="toggle-preventreadonscroll"
+							v-model="preventReadOnScroll"
+							type="checkbox"
+							class="checkbox">
+						<label for="toggle-preventreadonscroll">
+							{{ t('news', 'Disable mark read through scrolling') }}
+						</label>
+					</div>
+					<!---
+					<div>
+						<input id="toggle-compact"
+							v-model="compact"
+							type="checkbox"
+							class="checkbox">
+						<label for="toggle-compact">
+							{{ t('news', 'Compact view') }}
+						</label>
+					</div>
+					<div>
```

Comment:
Should all these be here if they're commented out?

Reply:
These are the actual options for the compact mode, which doesn't exist in 25.x. So it makes no sense to show this yet.

Reply:
In that case lets just remove it?